### ISSUE TITLE
Add a --fix-name-length option to align command output

### ISSUE THREAD

**Issue by syabro** [open-cli-tools/concurrently#417]
When using concurrently, providing custom names with the `-n` or `--names` option can cause output to become misaligned and difficult to read if the names have different lengths. This can be particularly problematic when the output is lengthy and spans multiple lines.

To address this issue, I propose adding a new `--fix-name-length` option that allows users to specify a fixed length for the command names. When this option is used, the command names will be padded with spaces or truncated as necessary to ensure that they all have the same length, and the output will be properly aligned.

For example, if a user runs the following command:

```
concurrently -n "API, TypeScript, VeryLong" --fix-name-length\
    "npm run api" "npm run typescript" "npm run somethingverylong"
```

The output would be aligned like this:

```
API        | output from npm run api
TypeScript | output from npm run typescript
VeryLong   | output from npm run somethingverylong
```

This would make it much easier to read and understand the output from each command, especially in cases where the output spans multiple lines.

I believe that adding a `--fix-name-length` option would be a valuable addition to concurrently, as it would improve the user experience for those who use custom names for their commands.

**Comment by gustavohenke** [comment]
I like this idea very much!

With one caveat: maybe it should just be `--pad-names`? Or something else that alludes to "padding"?
I feel like "fix" can seem like a bug is involved 😛

**Comment by syabro** [comment]
@gustavohenke sure! 
 `--pad-names` without any params would be great!

**Comment by gustavohenke** [comment]
🚢 This is now available in v9.0.0!
https://github.com/open-cli-tools/concurrently/releases/tag/v9.0.0
